backstage/templates/backend-secret.yaml

```diff
@@ -14,6 +14,8 @@ stringData:
   AUTH_MICROSOFT_CLIENT_SECRET: {{ .Values.auth.microsoft.clientSecret }}
   SENTRY_TOKEN: {{ .Values.auth.sentryToken }}
   ROLLBAR_ACCOUNT_TOKEN: {{ .Values.auth.rollbarAccountToken }}
+  GCP_CLIENT_EMAIL: {{ .Values.auth.gcpClientEmail }}
```

Comment:
There is nothing consuming this at the moment, should there be a reference in the `backstage/files/app-config.development.yaml.tpl` or in the values.yaml file in the form of `env: $secret ...` (something like line 232)?

I was thinking about not adding more things to this secret, consider this legacy and eventually remove it but maybe this will be handy for tests :+1:

Reply:
I added this to the app-config on the customer-common branch. I was mostly copying and pasting from the other secrets, kind of blindly.